CS_Ed_Week/CS_Ed_Week.md

```diff
@@ -76,6 +89,17 @@ A workshop for getting young people interested in computer science
       - list_name -> [1, 10, 2, 3, 4, 5]
     - Removing: list_name.remove(3)
       - list_name -> [1, 10, 2, 4, 5]
+### Printing lists and their elements
+- To print the entire contents of a list, simply supply the name of list as the argument to print()
+- To print an specific element of a list, supply the name of the list, followed by its index, to print()
+    - pets = ["dog", "cat", "bird"]
+    - print(pets)
+    - print(pets[2])
```

Comment:
You might want to add that lists and arrays start counting from 0 instead of 1. I don't think that's covered in the rest of the presentation